consensus/istanbul/core/vrank.go

```diff
@@ -85,19 +97,22 @@ func (v *Vrank) HandleCommitted(blockNum *big.Int) {
 		return
 	}
 
-	committedTime := v.TimeSinceStart()
-	if v.threshold > committedTime {
-		v.threshold = committedTime
-	}
-
-	vrankQuorumCommitArrivalTimeGauge.Update(int64(committedTime))
 	if len(v.commitArrivalTimeMap) != 0 {
 		sum := int64(0)
+		quorumCommitTime := time.Duration(0)
 		for _, v := range v.commitArrivalTimeMap {
```

Comment:
the variable 'v' may be confusing because this variable name is also used to indicate Vrank structure itself.
can we change the variable name to like 'arrivalTime'?

Reply:
Fixed in https://github.com/klaytn/klaytn/pull/1919/commits/20c3efdeda59931f37e62fd911ac607f53c6daf8